src/pgduckdb_background_worker.cpp

```diff
@@ -475,9 +481,11 @@ TriggerActivity(void) {
 	}
 
 	SpinLockAcquire(&BgwShmemStruct->lock);
-	auto state = GetState();
-	state->activity_count++;
-	SetLatch(state->latch);
+	auto state = FindState();
+	if (state) {
```

Comment:
Despite what I said (thought) in last PR, this can still happen. I can't reproduce manually, but the tests clearly hits it.

Reply:
Yeah I hit this yesterday when running manually. It happens when we trigger activity without the background worker having been started. The worst part is that was that it would then hold the spinlock forever. So let's also unlock the spinlock before we throw the error in GetState.